src/prefect/run_configs/local.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Iterable
+
+from prefect.run_configs.base import RunConfig
+
+
+class LocalRun(RunConfig):
+    """Configure a flow-run to run as a Local Process.
+
+    Note: The functionality here is experimental, and may change between
+    versions without notice. Use at your own risk.
+
+    Args:
+        - env (dict, optional): Additional environment variables to set for the process
+        - working_dir (str, optional): Working directory in which to start the
+            process, must already exist. If not provided, will be run in the
+            same directory as the agent.
+        - labels (Iterable[str], optional): an iterable of labels to apply to this
+            run config. Labels are string identifiers used by Prefect Agents
+            for selecting valid flow runs when polling for work
+
+    Examples:
+
+    Use the defaults set on the agent:
+
+    ```python
+    flow.run_config = LocalRun()
+    ```
+
+    Run in a specified working directory:
+
+    ```python
+    flow.run_config = LocalRun(working_dir="/path/to/working-directory")
+    ```
+
+    Set an environment variable in the flow run process:
+
+    ```python
+    flow.run_config = LocalRun(env={"SOME_VAR": "value"})
+    ```
+    """
+
+    def __init__(
+        self,
+        *,
```

Comment:
Why do you eat all the positional args?

Reply:
This doesn't "eat" the args, any arguments after `*,` in a python function are keyword only (https://www.python.org/dev/peps/pep-3102/). This prevents users from accidentally relying on positional ordering of the arguments (they all have to be passed in explicitly by name),  making future changes easier to make without breaking old user code.

Reply:
Oh cool! I haven't seen that pattern before. I assumed it was equivalent to `*_`